.github/workflows/pull-request.yml

```
@@ -187,6 +187,29 @@
+        uses: tj-actions/changed-files@v45
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.9
+      
+      - name: Run Python script
+        if: steps.changed-files.outputs.only_changed != 'true'
+        env: 
+          CHANGED_FILES: ${{ steps.changed-files.outputs.all_changed_files }}
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:69: Double quote to prevent globbing and word splitting

(shellcheck)